stylesheets/_component.datatables.scss

```diff
@@ -155,7 +174,8 @@ th.sorting_disabled:hover {
 .dataTables_info {
     display: none;
     float: left;
-    margin-top: 11px;
+    line-height: 30px;
+    margin-left: 12px;
```

Comment:
Can we move this onto `dt-buttons` to prevent info being misaligned when no buttons are present.

Reply:
obvs will need to be margin-right instead